package.json

```diff
@@ -8,7 +8,8 @@
         "postinstall": "yarn run buildAll",
         "heroku": "node server/lib/heroku.js",
         "buildAll": "yarn workspace @bible-engine/core run build && yarn workspace @bible-engine/importers run build && yarn workspace @bible-engine/server run generateApiClient && yarn workspace @bible-engine/server run build && yarn workspace @bible-engine/client run build",
-        "test:app": "yarn workspace @bible-engine/core run test"
+        "test:app": "yarn workspace @bible-engine/core run test",
+        "typeorm": "yarn ts-node ./node_modules/typeorm/cli.js"
```

Comment:
This is needed to run migration files directly without compiling

Normally, the typeorm CLI needs the migration files to be compiled to JS